packages/amplify-graphql-transformer-core/package.json

```diff
@@ -59,13 +59,15 @@
         "graphql-transformer-common": "4.23.0",
         "lodash": "^4.17.21",
         "md5": "^2.3.0",
+        "object-hash": "^3.0.0",
         "ts-dedent": "^2.0.0",
         "vm2": "^3.9.8"
     },
     "devDependencies": {
         "@types/fs-extra": "^8.0.1",
         "@types/md5": "^2.3.1",
-        "@types/node": "^12.12.6"
+        "@types/node": "^12.12.6",
+        "@types/object-hash": "^2.2.1"
```

Comment:
should this be on the same version as `object-hash`?

Reply:
2.2.1 is the latest version on @types/object-hash